client/validate.go

```diff
@@ -55,6 +57,7 @@ func newValidator(params validatorParams) *validator {
 		functionOutputFile:  params.outputFile,
 		stdoutFile:          defaultStdoutFile,
 		stderrFile:          defaultStderrFile,
+		envs: params.envs,
 	}
 
 	if !params.useBuildpacks {
```

Comment:
Can you make this work for the local function server too? (-buildpacks=false) mode

Reply:
Good point! Let me add it